mri/reconstructors/tests/test_reconstructors.py

```diff
@@ -0,0 +1,212 @@
+# -*- coding: utf-8 -*-
+##########################################################################
+# pySAP - Copyright (C) CEA, 2017 - 2018
+# Distributed under the terms of the CeCILL-B license, as published by
+# the CEA-CNRS-INRIA. Refer to the LICENSE file or to
+# http://www.cecill.info/licences/Licence_CeCILL-B_V1-en.html
+# for details.
+##########################################################################
+
+# Third-party import
+import numpy as np
+import unittest
+
```

Comment:
look at the blank lines section of pep8 for a better formatting in the future: https://www.python.org/dev/peps/pep-0008/#blank-lines